conferecebook/lib/EditProfile.dart

```diff
@@ -0,0 +1,646 @@
+    code = widget.code;
+    name = widget.name;
+    city = widget.city;
+    area = widget.area;
+    job = widget.job;
+    bio = widget.bio;
+    return WillPopScope(
+        onWillPop: () async => false,
+        child: Scaffold(
+          backgroundColor: const Color(0xffffffff),
+          body: Stack(
+            children: <Widget>[
+              Transform.translate(
+                offset: Offset(SizeConfig.screenWidth * 34.0,
+                    SizeConfig.screenHeight * 150.0),
+                child: Container(
+                  width: SizeConfig.screenWidth * 345.0,
+                  height: SizeConfig.screenHeight * 636.0,
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(30.0),
+                    color: const Color(0x80ececec),
+                  ),
+                ),
+              ),
+              Transform.translate(
+                offset: Offset(SizeConfig.screenWidth * 110.0,
+                    SizeConfig.screenHeight * 64.0),
+                child: Container(
+                  width: SizeConfig.screenWidth * 194.0,
+                  height: SizeConfig.screenHeight * 194.0,
+                  decoration: BoxDecoration(
+                    borderRadius:
+                        BorderRadius.all(Radius.elliptical(9999.0, 9999.0)),
+                    color: const Color(0xfff5f5f5),
+                  ),
+                ),
+              ),
+              Transform.translate(
+                offset: Offset(SizeConfig.screenWidth * 79,
+                    SizeConfig.screenHeight * 320.0),
+                child: SizedBox(
+                  width: SizeConfig.screenWidth * 260.0,
+                  child: Container(
+                    width: 260,
+                    height: 20,
+                    child: TextField(
+                    controller: cityController,
+                    onChanged: (String value) async {
+                        city = value;
+                    },
+                    decoration: InputDecoration(
+                      border: OutlineInputBorder(
+                          borderRadius:BorderRadius.circular(0.1)),
+                    ),
+                    style: TextStyle(
+                      fontFamily: 'Roboto',
+                      fontSize: 12,
+                      color: const Color(0xff1A2677),
+                      letterSpacing: 0.1875,
+                      fontWeight: FontWeight.w500,
+                      height: 1.2,
+                    ),
+                    textAlign: TextAlign.center,
+                  ),)
+                ),
+              ),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
+              Container(),
```

Comment:
Probably an error while merging conflits, needs to be corrected after merging with main!